src/components/cards/CreatePool/CreateActions.vue

```diff
@@ -0,0 +1,250 @@
+<script setup lang="ts">
+import { toRef, ref, toRefs, computed, ComputedRef, reactive } from 'vue';
+import PoolExchange from '@/services/pool/exchange/exchange.service';
+import { getPoolWeights } from '@/services/pool/pool.helper';
+// Types
+import { FullPool } from '@/services/balancer/subgraph/types';
+import {
+  TransactionReceipt,
+  TransactionResponse
+} from '@ethersproject/abstract-provider';
+import useTokenFiatMath from '@/composables/useTokenFiatMath';
+// Composables
+import useWeb3 from '@/services/web3/useWeb3';
+import useTransactions from '@/composables/useTransactions';
+import useEthers from '@/composables/useEthers';
+import { useI18n } from 'vue-i18n';
+import { dateTimeLabelFor } from '@/composables/useTime';
+import { useRoute } from 'vue-router';
+
+import useConfig from '@/composables/useConfig';
+import useTranasactionErrors, {
+  TransactionError
+} from '@/composables/useTransactionErrors';
+import useTokenApprovalActions from '@/composables/useTokenApprovalActions';
+import { Step, StepState, Action } from '@/types';
+import usePoolCreation from '@/composables/pools/usePoolCreation';
+
+/**
+ * TYPES
+ */
+type Props = {
+  tokenAddresses: string[];
+  amounts: string[];
+};
+
+type ActionState = {
+  init: boolean;
+  confirming: boolean;
+  confirmed: boolean;
+  confirmedAt: string;
+  error?: TransactionError | null;
+  receipt?: TransactionReceipt;
+};
+
+const defaultActionStateValues: ActionState = {
```

Comment:
Really minor, I'm almost sorry for bringing this up, but I don't think the variable name needs Values on the end? 😅 
```suggestion
const defaultActionState: ActionState = {
```